02/app.js

```diff
@@ -0,0 +1,55 @@
+//sprawdzenie loginu
+const spanMail = document.querySelector('.mail-error');
+const input = document.querySelector('input[name=login]');
+
+input.addEventListener('input', checkMail);
+
+function checkMail(e) {
+    const self = e.currentTarget;
+    const eMail = self.value;
+
+    if (!eMail.includes('@')) {
+        spanMail.innerText = 'Wprowadź poprawny adres e-mail.';
+        spanMail.style.color = 'red';
+    } else {
+        spanMail.innerText = 'Pole wypełnione poprawnie.';
+        spanMail.style.color = 'green';
+    }
+}
+//sprawdzanie haseł
+const inputPass1 = document.querySelector('input[name=pass1]');
+const inputPass2 = document.querySelector('input[name=pass2]');
+const labelPass1 = document.querySelector('label[for=formPass1]');
+const labelPass2 = document.querySelector('label[for=formPass2]');
+const form = document.querySelector('form');
+
+form.addEventListener('submit', checkPass);
+
+function checkPass(e) {
+    const pass1Self = e.target;
```

Comment:
Chyba brakuje `e.preventDefault()` - zastanawiam się jak to testowałaś bez tego :P